metadeploy/api/models.py

```diff
@@ -638,7 +636,7 @@ class Job(HashIdMixin, models.Model):
     enqueued_at = models.DateTimeField(null=True)
     job_id = models.UUIDField(null=True)
     status = models.CharField(choices=Status, max_length=64, default=Status.started)
-    org_id = models.CharField(null=True, blank=True, max_length=18)
+    org_id = models.CharField(max_length=18)
```

Comment:
@wlonk I wasn't sure why `org_id` would be optional for Jobs and Preflights...

Reply:
Making it not-null after it has been is a thorny migration. I would urge caution here.